src/Workspaces/Core/Portable/Workspace/Solution/Document.cs

```diff
@@ -200,6 +202,8 @@ public Task<SyntaxTree> GetSyntaxTreeAsync(CancellationToken cancellationToken =
 
         /// <summary>
         /// Gets the root node of the current syntax tree if it is available.
```

Comment:
Is this the only one left that uses the ambiguous "if it is available" wording? Can we update this one too?